### ISSUE TITLE
Two separate websites, same library version, same browser version, different fingerprint.

### ISSUE THREAD

**Issue by bazaarcodinglabs** [Valve/fingerprintjs2#185]
I have two separate websites, website A and website B, each using the same version of the fingerprint2 library.  Using the same browser, I have noticed that the fingerprint was different on website A.  It is different because one font ("Wingdings 3") was missing in the 'js_fonts' array in website A.  The font was not being detected because on website A, the "line-height" on website A was explicitly set to 14px for all divs - this is being done on our internal stylesheet.  

I found the solution by resetting the "line-height" on the span to "normal", this allows the "line-height" to be the actual height of a particular font, rather than allowing it to inherit the line-height from another style.

The solution is as follows, its a one liner (line 12), could you consider including it in your library please?

```

1.                 // creates a span where the fonts will be loaded
2.                 var createSpan = function() {
3.                     var s = document.createElement("span");
4.                     /*
5.                      * We need this css as in some weird browser this
6.                      * span elements shows up for a microSec which creates a
7.                      * bad user experience
8.                      */
9.                      s.style.position = "absolute";
10.                     s.style.left = "-9999px";
11.                     s.style.fontSize = testSize;
12.                     s.style.lineHeight = "normal";
13.                     s.innerHTML = testString;
14.                     return s;
15.                 };

```

Many thanks.

Barry Edwards

**Comment by Valve** [comment]
Yes, of course, please do a PR and I'll merge it

**Comment by bazaarcodinglabs** [comment]
Sure, will do tomorrow.  Thank you.

**Comment by Valve** [comment]
Fixed in #193
